types/is-hotkey/index.d.ts

```diff
@@ -4,6 +4,10 @@
 //                 Kalley Powell <https://github.com/kalley>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+import * as React from 'react';
```

Comment:
The react typings are quite heavy and churn a lot, but this library surface is very small. We also need to consider things like preact or other libraries that might be imported in a way that would create conflict with react. I think this dependency needs to be removed and replaced with an inline structural match for `React.KeyboardEvent`

Reply:
Thanks for the suggestion! I've removed React as dependency